extra_data/reader.py

```diff
@@ -629,6 +631,10 @@ def _merge_aliases(self, alias_dicts):
 
         return new_aliases
 
+    def _merge_alias_files(self, *alias_files):
+        all_files = chain.from_iterable(alias_files)
+        return list(set(all_files))
```

Comment:
```suggestion
        return sorted(set(all_files))
```

Sets give a random order, which doesn't really matter, but it's always nice to be deterministic if possible.

Reply:
Fixed in d92ccd1.